### PR TITLE
Verify ipcamera interaction

### PR DESCRIPTION
**Pull request by bvanwijk**
I have verified all the different calls from the backend to the camera from polycast are working.
Where necessary I have changed the calls. 
The response of every request to the camera is now also verified.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/camera/CameraController.java [hunk 1]
```diff
@@ -76,18 +72,11 @@ private StreamController getStreamController() {
   }
 
   /**
-   * Sets up the logger.
+   * Returns the single instance of the logger.
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/camera/CameraController.java]: Maybe this should be singleton instead of single

**Comment by jochem725** [comment]
Seems fine, I've added a small comment regarding a comment, but I think this can be merged.

**Comment by NaomidR** [comment]
Looks good, great that all the calls are working now. I agree with merging this.

File: backend/src/main/java/com/benine/backend/camera/BasicCamera.java [hunk 2]
```diff
@@ -1,12 +1,7 @@
...
-
-import com.benine.backend.Preset;
 import com.benine.backend.video.StreamType;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-
 /**
  * Interface for communication with remote camera's.
  * @author Bryan
```
Comment (egedebruin) [inline backend/src/main/java/com/benine/backend/camera/BasicCamera.java]: We are working as a team so you should replace Bryan with Benine or remove your name.

**Comment by egedebruin** [comment]
Looks very good, I agree with merging this.

**Comment by Kingdorian** [comment]
Looks good, I think we can merge this but I think it would be good to wait for @jochem725's pulrequest to  be merged into develop (merge conflict wise).

File: backend/src/main/java/com/benine/backend/camera/BasicCamera.java (hunk 2 above)
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/camera/BasicCamera.java]: Well noticed I will remove it.

**Pull request merged**